### PR TITLE
mgr/rook: misc fixes for 'orch ps'

### PR DESCRIPTION
**Pull request by liewegas**
```
NAME   HOST  STATUS         REFRESHED  AGE  VERSION    IMAGE NAME                    IMAGE ID   CONTAINER ID                      
mgr.a  m01   running (13s)  0s ago     13s  <unknown>  quay.io/liewegas/ceph:latest  <unknown>  rook-ceph-mgr-a-78f7668674-d8jl5  
mon.a  m01   running (41m)  0s ago     41m  <unknown>  quay.io/liewegas/ceph:latest  <unknown>  rook-ceph-mon-a-557b498bc7-hv6hq  
```

### CODE REVIEW

**Comment by tchaikov** [comment]
```
[ RUN      ] TestMockMirrorDisableRequest.Success
/home/jenkins-build/build/workspace/ceph-pull-requests/src/common/mutex_debug.h: In function 'ceph::mutex_debug_detail::mutex_debug_impl<<anonymous> >::~mutex_debug_impl() [with bool Recursive = false]' thread 7fb9b7c62700 time 2020-03-10T23:53:58.426027+0000
/home/jenkins-build/build/workspace/ceph-pull-requests/src/common/mutex_debug.h: 109: FAILED ceph_assert(r == 0)
 ceph version Development (no_version) octopus (rc)
 1: (ceph::__ceph_assert_fail(char const*, char const*, int, char const*)+0x19d) [0x7fb9ca8d9867]
 2: (()+0x1598ae9) [0x7fb9ca8d9ae9]
 3: (ceph::mutex_debug_detail::mutex_debug_impl<false>::~mutex_debug_impl()+0x31) [0x5624b9dd2deb]
 4: (()+0x1e4e7a8) [0x5624ba5547a8]
 5: (()+0x1e511c9) [0x5624ba5571c9]
 6: (ContextWQ::process(Context*)+0xdd) [0x5624b9f8486b]
 7: (ThreadPool::PointerWQ<Context>::_void_process(void*, ThreadPool::TPHandle&)+0x32) [0x5624b9fe9396]
 8: (ThreadPool::worker(ThreadPool::WorkThread*)+0x730) [0x7fb9ca899544]
 9: (ThreadPool::WorkThread::entry()+0x32) [0x7fb9ca89d4d2]
 10: (Thread::entry_wrapper()+0x78) [0x7fb9ca876cdc]
 11: (Thread::_entry_func(void*)+0x18) [0x7fb9ca876c5a]
 12: (()+0x76db) [0x7fb9d42416db]
 13: (clone()+0x3f) [0x7fb9c82b088f]
/home/jenkins-build/build/workspace/ceph-pull-requests/src/test/run-rbd-unit-tests.sh: line 20: 10379 Aborted                 (core dumped) RBD_FEATURES=$i unittest_librbd

```

**Comment by liewegas** [comment]
retest this please

**Comment by liewegas** [comment]
@dillaman ^ seen that one?

**Comment by dillaman** [comment]
> @dillaman ^ seen that one?

Negative -- I've been watching the jenkins failures recently and I really only see the random `unittest_rbd_mirror` failure where the logs are cut off.

File: src/pybind/mgr/rook/rook_cluster.py [hunk 1]
```diff
@@ -312,18 +313,32 @@ def predicate(item):
...
                 "hostname": d['spec']['node_name'],
                 "labels": d['metadata']['labels'],
-                'phase': d['status']['phase']
+                'phase': d['status']['phase'],
+                'container_image_name': image_name,
+                # note: we want UTC but no tzinfo
+                'created': d['metadata']['creation_timestamp'].astimezone(
+                    tz=datetime.timezone.utc).replace(tzinfo=None),
+                'started': d['status']['start_time'].astimezone(
+                    tz=datetime.timezone.utc).replace(tzinfo=None),
```
Comment (sebastian-philipp) [inline src/pybind/mgr/rook/rook_cluster.py]: those might be missing in the json, if the container are not schedulable

**Comment by liewegas** [comment]
retest this please

File: src/pybind/mgr/rook/rook_cluster.py [hunk 2]
```diff
@@ -312,19 +313,38 @@ def predicate(item):
...
+                break
+
+            s = {
                 "name": d['metadata']['name'],
                 "hostname": d['spec']['node_name'],
                 "labels": d['metadata']['labels'],
-                'phase': d['status']['phase']
-            })
+                'phase': d['status']['phase'],
+                'container_image_name': image_name,
```
Comment (jmolmo) [inline src/pybind/mgr/rook/rook_cluster.py]: 'container_image_id':  image_id

**Pull request merged**